src/backend/vulkan/src/conv.rs

```diff
@@ -148,6 +148,203 @@ pub fn map_format(surface: format::SurfaceType, chan: format::ChannelType) -> Op
     })
 }
 
+pub fn map_vk_format(
+    fmt: vk::Format
+) -> Option<(format::SurfaceType, format::ChannelType)> {
+    use ash::vk::Format::*;
+    use core::format::SurfaceType::*;
+    use core::format::ChannelType::*;
+    match fmt {
```

Comment:
it's easier to write `Some(match fmt {` here, and then only `return None` in some of the branches